addons/purchase_stock/models/stock_rule.py

```diff
@@ -16,14 +16,17 @@ class StockRule(models.Model):
     def _get_message_dict(self):
         message_dict = super(StockRule, self)._get_message_dict()
         message_dict.update({
-            ('buy', False, True): _('When products are needed in %s, <br/> a request for quotation is created to fulfill the need.') % (self.location_id.display_name,)
+            ('buy', False, True): _('When products are needed in <b><i>%s</i></b>, <br/> a request for quotation is created to fulfill the need.') % (self.location_id.display_name,)
```

Comment:
@dat-odoo shouldn't you adapt `_search_rule` to handle the global rules?
also please refer to rev https://github.com/odoo/odoo/commit/b1b3de1339a13504ac5e223aec465bd70417d11e#diff-23b5396fd047e334041d578a52ace527 in your commit message